include/darwin/hax_interface_mac.h

```diff
@@ -50,6 +50,7 @@
 #define HAX_VM_IOCTL_SET_RAM _IOWR(0, 0x82, struct hax_set_ram_info)
 #define HAX_VM_IOCTL_VCPU_DESTROY _IOR(0, 0x83, uint32_t)
 #define HAX_VM_IOCTL_ADD_RAMBLOCK _IOW(0, 0x85, struct hax_ramblock_info)
+#define HAX_VM_IOCTL_SET_RAM2 _IOWR(0, 0x86, struct hax_set_ram_info2)
```

Comment:
`_IOW` would be sufficient, since all parameters are currently write-only from the caller's point of view. But it's possible that one day we will want to return some data back to user space in a reserved field, so `_IOWR` is fine.